src/pages/patientView/genomicOverview/GenomicOverview.tsx

```diff
@@ -118,6 +385,12 @@ export default class GenomicOverview extends React.Component<
         );
     }
 
+    componentWillReceiveProps(nextProps: Readonly<IGenomicOverviewProps>) {
```

Comment:
this is so it doesn't render if locus hasn't actually changed?

Reply:
Right. This is to prevent resetting zoom state on window resize. Hopefully this didn't introduce some unknown side effects.

Reply:
We should put a comment on this to explain